frontend/src/app/supportAdmin/PendingOrganizations/modals/VerifyPendingOrganizationConfirmationModal.tsx

```diff
@@ -0,0 +1,87 @@
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import React from "react";
+import Modal from "react-modal";
+
+import Button from "../../../commonComponents/Button/Button";
+import { PendingOrganization } from "../../../../generated/graphql";
+
+interface VerifyConfirmationModalProps {
+  onVerifyConfirm: () => void;
+  onClose: () => void;
+  setVerifyConfirmation: (verify: boolean) => void;
+  organization: PendingOrganization;
+  isLoading: boolean;
+}
+
+const VerifyPendingOrganizationConfirmationModal: React.FC<
+  VerifyConfirmationModalProps
+> = ({
+  onVerifyConfirm,
+  onClose,
+  setVerifyConfirmation,
+  organization,
+  isLoading,
+}) => {
+  return (
+    <Modal
+      isOpen={true}
```

Comment:
Can you use the `isOpen` prop to control the display of the modal instead of conditional rendering?. Reason for asking is that after a certain version (can't remember which one) the react modal libray throws a warning when conditional rendering is used. Let me know if you need more details on what I am talking about and I am happy to pair.

Reply:
ooo thank you!! I'll fix that up!